plugins/Monolog/Formatter/LineMessageFormatter.php

```diff
@@ -62,6 +82,27 @@ public function format(array $record)
         return $total;
     }
 
+    private function jsonMessage($class, $message, $date, $record)
+    {
+        $trace = isset($record['context']['trace']) ? self::formatTrace($record['context']['trace']) : '';
+        $requestId = isset($record['extra']['request_id']) ? $record['extra']['request_id'] : '';
+
+        $message = [
+            "tag" => $class,
+            "datetime" => $date,
+            "message" => $message,
+            "level" => $record['level_name'],
+            "trace" => $trace,
+            "requestId" => $requestId
+        ];
+
+        if (gettype($this->customFunction) === "string") {
```

Comment:
this might check for not being empty and also if it's actually callable
```suggestion
        if (is_callable($this->customFunction)) {
```